WORKSPACE.bazel

```diff
@@ -1,12 +1,15 @@
 workspace(name = "com_github_3rdparty_eventuals_grpc")
 
 load("//bazel:repos.bzl", "repos")
+
```

Comment:
Why the newlines in this file?

Reply:
Yep. This is how buildifier formats this file:)